cfme/tests/automate/__init__.py

```diff
@@ -86,3 +86,6 @@ def fill_dialog_field(list)
     $evm.log(:info, "Tag exists: #{tag_values[1]}")
     """
 )
+
+imported_domain_info = {"domain": "testdomain", "namespace": "test", "klass": "TestClass1",
+                        "method": "meh", "script": '$evm.log(:info, ":P")'}
```

Comment:
@ganeshhubale as per my opinion. In current PR we can hardcore this but with one task :) **Refactor: test_git_import.py**

I found that every one using own repository in this file. I think we can collect this all stuff in single ropos with different branch. I would prefer `https://github.com/RedHatQE/ManageIQ-automate-git.git` as main repo. 

Add repo info (link and required data like branch, tag, namespace etc...) in our `data yaml`.

Reply:
@mshriver what you think about collecting all stuff in `ManageIQ-automate-git` repo. as everyone hard-coded own things.

Reply:
@digitronik @ganeshhubale This is kind of what I was getting at above. I think what you're suggesting here is a good solution to this problem.